hack/jenkins/e2e.sh

```diff
@@ -34,6 +34,11 @@ if [[ "${CIRCLECI:-}" == "true" ]]; then
     WORKSPACE=`pwd`
 else
     # Jenkins?
+    # Copy GCE keys so we don't keep cycling them.
+    mkdir -p ${WORKSPACE}/.ssh/
+    cp /var/lib/jenkins/gce_keys/google_compute_engine ${WORKSPACE}/.ssh/
```

Comment:
I added a string of comments above it about the setup. Does this look all right?